### DISCUSSION TITLE
Hiding the folders

### DISCUSSION THREAD

**Discussion by Kitosight** [microsoft/vscode-discussions#550] [Extension Development QnA]
So, I hid one of my folders in another folder (aka workspace). But when I open it, the folder is still showing.
How can I do the same just like the `.git` folders?

![image-165.png](https://user-images.githubusercontent.com/70456337/228924693-9d1162d0-2035-4abf-9c42-b7cfb0cbe0fa.png)

![image-103.png](https://user-images.githubusercontent.com/70456337/228924689-4c75496e-36bc-412f-916a-16e688150e25.png)

![image-78.png](https://user-images.githubusercontent.com/70456337/228924690-49e80b1f-abfc-4aaa-8a66-f0929d365df5.png)

![image-100.png](https://user-images.githubusercontent.com/70456337/228924688-d75e40c0-a62c-4cc2-9d39-360e57cf472f.png)

**Comment by olguzzar** [comment]
The files/folders that are hidden in the explorer is based on the Files: Exclude setting. You can add a new pattern to that setting to hide your .workspace-data folder
![files exclude](https://user-images.githubusercontent.com/95261576/229520797-71b48c73-f9dc-43ba-8456-46e08c315b2f.png)

As a note, please use this discussion forum for extension-specific questions. I'm going to close this discussion but feel free to reopen if you have an extension-related question on this topic

**Reply by Kitosight** [reply]
> As a note, please use this discussion forum for extension-specific questions. I'm going to close this discussion but feel free to reopen if you have an extension-related question on this topic

It's related to my own extension, and one of its features is a (custom) workspace creation (I don't really know how to call it, so let's call it so).
How can I add a pattern using my extension after a workspace creation?
